examples/db-templates/mongodb-ephemeral-template.json

```diff
@@ -152,6 +160,11 @@
   ],
   "parameters": [
     {
+      "name": "DEPLOYMENT_LIMIT_SIZE",
```

Comment:
I don't like POD_MEMORY_LIMIT.  I might buy MEMORY_LIMIT.  Or MAX_MEMORY.  And you need a display name on these parameters.